### ISSUE TITLE
Add compatibility with Internet Game Database API version 2.0

### ISSUE THREAD

**Issue by aloupfor** [igdb/igdb-api-php#10]
Hello,

I got an API key from from `mashape.com` and tried the examples from the `igdb-api-php/examples/` directory and got errors such as : 

```
Client error: `GET https://www.igdb.com/api/v1/games?offset=0&limit=25` 
resulted in a `401 Unauthorized` response:
HTTP Token: Access denied.

500 Internal Server Error - InvalidAPiKeyException
1 linked Exception: ClientException
```

From what I understood, the Internet Game Database API is now in version 2.0 since May 2016 (when you created it on `mashape.com`). I assume this from the differences between the documentations here : https://www.igdb.com/api/v1/documentation and here : https://market.mashape.com/igdbcom/internet-game-database

I was confused since : 
- both url have `v1` that I thought was for version 1 (https://www.igdb.com/api/v1 and https://igdbcom-internet-game-database-v1.p.mashape.com/)
- the documentation on `mashape.com` does not state clearly that the API is now in 2.0. The only place I found an information about this new version is here : https://www.igdb.com/users/aloupfor/api_key which states "API 2.0".

Seeing the errors I got and the code (I found in an annotation : "The base URL to the API v1."), I guess that `igdb/igdb-api-php` was build to work with the API 1.0 and that no api key are provided for this version anymore.

Will there be a version of `igdb/igdb-api-php` for the 2.0 API?

Best regards.

aloupfor

**Comment by euantorano** [comment]
Yeah, the changes I made to this library were aimed at v1. I've been busy, so I didn't even notice there was a v2 API (nor that the guys at IGDB hadn't updated this).

If I get a chance, I'll look at updating this library to v2.

**Comment by sharpless512** [comment]
@euantorano thanks for the good answer!

@aloupfor If you still want to use the API please use https://api.igdb.com/ not mashape. We will terminate the mashape gateway at the end of the month
